XForm/XForm/Columns/ArrayColumn.cs

```diff
@@ -22,6 +22,7 @@ public ArrayColumn(XArray allValues, ColumnDetails columnDetails)
         {
             _allValues = allValues;
             ColumnDetails = columnDetails;
+            SetSelector(allValues.Selector);
```

Comment:
Hmm. This is a bad sign. Anything which returns an ArrayColumn should be calling SetSelector in Next, before you call one of the getters. We should probably throw in each getter if the selector hasn't been set instead, and then debug the case where you see this failing. I can help with the debugging if you need.